srcpkgs/xmobar/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'xmobar'
+pkgname=xmobar
+version=0.27
+revision=1
+build_style=haskell-stack
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXinerama-devel libXrandr-devel \
+	     $(vopt_if xft libXft-devel) \
```

Comment:
intend with only one sapce. Also no need of escaping a newline insiede quotations.

Reply:
done